client/pdo/client/scripts/UpdateCLI.py

```diff
@@ -57,6 +57,27 @@ def __next__(self) :
         except EOFError as e :
             raise StopIteration
 
+def ReadEserviceDatabase(eservice_db_file):
+    """ Read the eservice data base. The db is a json file. 
+    Key is enclave id , value is eservice URL. The db contains the info for all enclaves known to the client.
+    Return the database as a dictonary. 
+    
+    This method is used by the client to get URLs of the enclaves listed in the pdo file.
+    """
+
+    if os.path.exists(eservice_db_file):
+        try:
+            with open(eservice_db_file, 'r') as fp:
+                eservice_db = json.load(fp)
+        except Exception as e:
+            logger.error('Could not open eservice database file : ' + str(e))
+            sys.exit(-1)
+    else:
+        logger.error('Attempting to read non-existent eservice data base file.')
+        sys.exit(-1)
+    
+    return eservice_db
+
 ## -----------------------------------------------------------------
```

Comment:
again... why not push this into pdo/service_client/enclave.py?

Reply:
the Updatedatabase and Readdatabase methods have been moved to client/controller/commands/eservice.py. It will be good to have shell commands for updating the database, if the client provides the database name, and the service urls,